### ISSUE TITLE
keys cmd: implement keys list or keys ls

### ISSUE THREAD

**Issue by dan-da** [maidsafe/safe-api#392]
Currently I can find no way to list safe keys.

I think that `keys list` or `keys ls` would solve this.

related:  there should be a a way to view/dump private and public keypair for each safe key, or only pubkeys.

**Comment by bochaco** [comment]
SafeKeys linked form a Wallet can be listed using the `cat` command on a Wallet URL, you'll get the list of SafeKeys URLs and friendly name assigned to them, we could show pk/sk there too.

Listing other SafeKeys which are not linked from a Wallet, then it's related to this comment: https://github.com/maidsafe/safe-api/issues/391#issuecomment-576725096

**Comment by dan-da** [comment]
yes, I'd like to be able to see the pk/sk somehow.  for dev/debug purposes but also for backup and general mucking about with keys.
